contracts/TokenBridgeBaseUpgradable.sol

```diff
@@ -36,4 +39,6 @@ abstract contract TokenBridgeBase is NonblockingLzApp, ReentrancyGuard {
 
     /// @dev Overrides the renounce ownership logic inherited from openZeppelin `Ownable`
     function renounceOwnership() public override onlyOwner {}
-}
\ No newline at end of file
+
+    function _authorizeUpgrade(address newImplemantation) internal override onlyOwner {}
```

Comment:
Can you add the docs from [here](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v4.9.3/contracts/proxy/utils/UUPSUpgradeable.sol#L98):
```
@dev Function that should revert when `msg.sender` is not authorized to upgrade the contract. Called by
     * {upgradeTo} and {upgradeToAndCall}.
```